testing/regress/ecl/setup/files.ecl

```diff
@@ -62,6 +63,10 @@ SHARED STRING _indexPrefix := '~regress::' +
   EXPORT filePrefix := #IFDEFINED(root.filePrefix, _filePrefix);
   EXPORT indexPrefix := #IFDEFINED(root.filePrefix, _indexPrefix);
 #end
+
+EXPORT QueryFilePrefixId := __PLATFORM__ + '::' + Str.ToLowerCase(WORKUNIT) + '::';
```

Comment:
You could use /_/_TARGET/_PLATFORM/_/_ here (a constant at parse time).  Possibly more correct since it is the platform you try and run on rather than the one that actually runs it, and it would avoid the problem fixed by the other PR.

Reply:
Will this prefix cause problems on a system with file scope checking enabled.  Is it better not to have :: after the workunit (something else instead) to avoid introducing another scope?

Reply:
> You could use __TARGET_PLATFORM__ here (a constant at parse time

Ok, sounds good, didn't know about that constant. I will change to use that.

Reply:
>Will this prefix cause problems on a system with file scope checking enabled. Is it better not to have :: after the workunit (something else instead) to avoid introducing another scope?

scope checking would have to allow access to the the root scope for the regression suite output (~regress), can't imagine there'd be any further scope permissions below that, so can't really see how scope checking could be an issue here.

Using a unique scope vs. a unique named file, probably not a big deal, but Dali's implementation means it's more efficient to have unique scopes than lots of unique names in the same scope.